.github/pull_request_template.md

```diff
@@ -2,7 +2,11 @@
 
 These changes Fix # | allow the user to | improve | ...
 
+<<<<<<< HEAD:.github/PULL_REQUEST_TEMPLATE/feature.md
 ### Things to add, update or check before this PR can be merged.
+=======
```

Comment:
Are these artifacts from merge conflicts or is that some special markdown syntax for the github templates?
If it is the former, please make sure to remove them.